### ISSUE TITLE
Is there a pre-trained non-robust model file?

### ISSUE THREAD

**Issue by ndb796** [MadryLab/constructed-datasets#9]
Hi, thank you for sharing good project :)

Can you share a pre-trained model file that trained on the non-robust dataset? (88% accuracy, 0% robust accuracy)

I could easily find the adversarially trained model file (robust model) in various repositories.
However, I couldn't find the non-robust model file...

**Comment by ndb796** [comment]
Using ResNet-18 architecture, I reproduced various model files including models that trained on the robust dataset and non-robust dataset in this repository.
Although my reproduced accuracies are a bit lower than the accuracies in this original paper, I shared my source codes and models.

https://github.com/ndb796/Pytorch-Adversarial-Training-CIFAR

**Comment by andrewilyas** [comment]
Thanks for this link! If you are interested, I can add a link to this repository from our README as an independent reproduction!

**Comment by ndb796** [comment]
If you link to my repository from your README, of course, it would be an honor, Thank you!
I think it's a really interesting and nice paper.

**Comment by andrewilyas** [comment]
Done! Closing this issue now.
